framework/src/application/schema/application_config_schema.js

```diff
@@ -326,6 +326,24 @@ module.exports = {
 												type: 'string',
 												format: 'publicKey',
 											},
+											hashOnion: {
```

Comment:
Hashes should not be added directly to the application config. Our commander generates a JOSN file, and in application config, we should just provide the path to that file. It's up to the user to generate that file with the commander or any other tool. Adding the config directly to application config JSON will make it more coupled and complex.

Reply:
Currently we are expecting the forger to put their info under `forging` (encryptedPassphrase and the public key). Also, encryptedPassphrase is generate by commander or any other tool too.
I don't think it make sense to separate them because both of the information are required when forging.
Maybe it make sense to split the forging information into other config, but let's discuss that separately